test/Microsoft.AspNet.Mvc.Xml.Test/XmlAssert.cs

```diff
@@ -42,8 +70,24 @@ private static XElement SortAttributes(XElement element)
         {
             return new XElement(
                     element.Name,
+                    element.Nodes().OfType<XText>().FirstOrDefault(),
                     element.Attributes().OrderBy(a => a.Name.ToString()),
                     element.Elements().Select(child => SortAttributes(child)));
         }
+
+        // Since ToString() on an XDocument gives a formatted string making it difficult
+        // to view differences in strings when a test fails, here the formatting is disabled.
+        private static string GetRawXml(this XDocument xdocument)
```

Comment:
- again, do not make this an extension method.  call `GetRawXml(argument)` instead
- don't abbreviate; just call the parameter `document`